lib/runner/TestRunner.js

```diff
@@ -57,12 +57,14 @@ module.exports = class TestRunner {
         }
     }
 
-    async runSuiteForLocale(testSuite, context, locale) {
+    async runSuiteForLocale(testSuite, context, locale, configurationOverride) {
+        // This may have already been called, but needs to be called again inside of the Jest runner
+        await Configuration.configure(this._config);
```

Comment:
if the intent of the pr is to overwrite the configuration, is this still needed?